index.js

```diff
@@ -28,14 +30,18 @@ module.exports = (oppiabot) => {
     // the whitelisted accounts.
     if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
       await apiForSheetsModule.checkClaStatus(context);
-      await checkPullRequestLabelsModule.checkChangelogLabel(context);
-      await checkPullRequestBranchModule.checkBranch(context);
-      await checkWIPModule.checkWIP(context);
+      if (!reposWithOnlyCLACheck.includes(context.repo().repo.toLowerCase())) {

```

Comment:
Rather than having a flag that disables all other functionality (sort of like a blacklist), could we instead use a 'whitelist' approach wherein we have a list of features enabled for one repository versus another? The initial state for the Oppia repository might be to enable everything, and for Oppia-Android might just be the CLA check. However, we definitely will want more features in the future, some of which won't be enabled for oppia/oppia but will for oppia/oppia-android. I think we should build this with those future uses in mind.

Reply:
Done.